<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parallel Prefix Sum Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="QGroupBox" name="fixedSizeGroup">
          <property name="title">
            <string>Program s pevnou velikostí pole</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout1">
            <item>
              <widget class="QPushButton" name="computeFixedButton">
                <property name="text">
                  <string>Počítať</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QTextEdit" name="fixedGeneratedTextEdit">
                <property name="placeholderText">
                  <string>Generované prvky</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QTextEdit" name="fixedResultTextEdit">
                <property name="placeholderText">
                  <string>Výsledok</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="scalableSizeGroup">
          <property name="title">
            <string>Škálovateľný program s premenlivou veľkosťou poľa</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout2">
            <item>
              <widget class="QLabel" name="elementsLabel">
                <property name="text">
                  <string>Počet prvkov:</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QSpinBox" name="elementsCountSpinBox">
                <property name="minimum">
                  <number>1</number>
                </property>
                <property name="maximum">
                  <number>10000</number>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="threadsLabel">
                <property name="text">
                  <string>Počet vlákien:</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QSpinBox" name="threadsCountSpinBox">
                <property name="minimum">
                  <number>1</number>
                </property>
                <property name="maximum">
                  <number>32</number>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="computeScalableButton">
                <property name="text">
                  <string>Počítať</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QTextEdit" name="scalableGeneratedTextEdit">
                <property name="placeholderText">
                  <string>Generované prvky</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QTextEdit" name="scalableResultTextEdit">
                <property name="placeholderText">
                  <string>Výsledok</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
